### PR TITLE
Update allowselfservicepurchase-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added an additional note about running the command individually fr each value if there are multiple values for $product

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5757

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 1]
```diff
@@ -135,6 +135,14 @@ $product = Get-MSCommerceProductPolicies -PolicyId AllowSelfServicePurchase | wh
 Update-MSCommerceProductPolicy -PolicyId AllowSelfServicePurchase -ProductId $product.ProductID -Enabled $false
 ```
 
+If there are multiple values for the product, you can run the command individually for each value like this.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
If there are multiple values for the product, you can run the command individually for each value as follows:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**